examples/bloom_filter/host_bulk_arrow_policy_example.cu

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2024, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cuco/bloom_filter.cuh>
+
+#include <thrust/count.h>
+#include <thrust/device_vector.h>
+#include <thrust/execution_policy.h>
+#include <thrust/sequence.h>
+
+#include <iostream>
+
```

Comment:
```suggestion

/**
 * @file arrow_policy_example.cu
 * @brief Demonstrates usage of an arrow-compatible bloom filter
 *
 * In addition to the default policy aimed at achieving the speed of light 
 * performance on the device, `cuCollections` offers an `arrow_filter_policy`
 * that allows users to easily create a bloom filter that mimics the behavior
 * of the bloom filter defined in Apache Arrow:
 * https://github.com/apache/arrow/blob/be1dcdb96b030639c0b56955c4c62f9d6b03f473/cpp/src/parquet/bloom_filter.cc#L219-L230.
 *
 * @note This example is for demonstration purposes only. It is not intended to show the most
 * performant way to do the example algorithm.
 */
 
```

Reply:
To highlight our hassle-free usage of the arrow filter.  Let's also rename the file as `arrow_policy_example.cu`.